src/runtimes/python3/bootstrap.ts

```diff
@@ -0,0 +1,26 @@
+import { join } from 'path';
+import { spawn } from 'child_process';
+
+// `PYTHONPATH` is *not* a restricted env var, so only set the
+// default one if the user did not provide one of their own
+if (!process.env.PYTHONPATH) {
+	process.env.PYTHONPATH = process.env.LAMBDA_RUNTIME_DIR;
+}
+
+let pythonBin = 'python3';
+const fallback = () => {
+	pythonBin = 'python';
+};
+const child = spawn(pythonBin, ['--version', '2>&1']);
+child.on('error', fallback);
+child.stdout.on('data', (data?: string) => {
```

Comment:
Python prints the version to stderr.

```suggestion
child.stderr.on('data', (data?: string) => {
```

Reply:
I used the same `2>&1` but maybe I did it wrong 🤷‍♂

Reply:
`2>&1` isn't going to work directly with `spawn()`, is a syntax specific to shell scripts.

Reply:
Here's a very old issue talking about how to achieve this in Node, but I don't think it was ever implemented: https://github.com/nodejs/node-v0.x-archive/issues/5831

Reply:
Ah yes. Fixed!